scripts/ansible/roles/linux/docker/vars/redhat/oopta.yml

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) Open Enclave SDK contributors.
```

Comment:
I think here is a mistype and should be named ootpa.yml

Reply:
Oh, absolutely! Good catch, thanks!